[
  {
    "id" : "8aed4643-bf2e-459f-a9d1-d4cd336f4f55",
    "prId" : 55986,
    "prUrl" : "https://github.com/ansible/ansible/pull/55986#pullrequestreview-261262792",
    "prSource" : "GitHub",
    "comments" : [
      {
        "id" : "ab4f46fa-0c6a-4300-82b7-18cda43565eb",
        "parentId" : null,
        "authorId" : "4ffac6b8-d7eb-449a-8c43-463d9fd5587a",
        "body" : "I think it's not pylint itself, but our own checks...",
        "createdAt" : "2019-07-12T10:13:43Z",
        "updatedAt" : "2019-07-12T18:38:31Z",
        "lastEditedBy" : "4ffac6b8-d7eb-449a-8c43-463d9fd5587a",
        "tags" : [
        ]
      },
      {
        "id" : "008531de-aa15-4817-87b2-8e7ea5a017de",
        "parentId" : "ab4f46fa-0c6a-4300-82b7-18cda43565eb",
        "authorId" : "838eb58c-0fa8-4742-994f-a407c7183e57",
        "body" : "It is pylint.  There are a variety of bugs opened against pylint wrt namespace modules.  One has been fixed but others remain open including for this test.",
        "createdAt" : "2019-07-12T13:40:21Z",
        "updatedAt" : "2019-07-12T18:38:31Z",
        "lastEditedBy" : "838eb58c-0fa8-4742-994f-a407c7183e57",
        "tags" : [
        ]
      }
    ],
    "commit" : "6abd71470c0c53987ad7ec902a4e3a899df378f0",
    "line" : 21,
    "diffHunk" : "@@ -1,1 +19,23 @@from ansible.module_utils._text import to_bytes\n\n# Pylint doesn't understand Python3 namespace modules.\nfrom ..change_detection import update_file_if_different  # pylint: disable=relative-beyond-top-level\nfrom ..commands import Command  # pylint: disable=relative-beyond-top-level"
  }
]